src/bootstrap/dist.rs

```diff
@@ -821,6 +821,11 @@ impl Step for Src {
                 // not needed and contains symlinks which rustup currently
                 // chokes on when unpacking.
                 "library/backtrace/crates",
+                // these are 30MB combined and aren't necessary for building
+                // the standard library.
+                "library/stdarch/crates/Cargo.toml",
```

Comment:
This should have been
```
"library/stdarch/Cargo.toml",
```
I think. In Rust 1.61.0, `rust-src` includes `src/rust/library/stdarch/Cargo.toml`, which includes
```toml
[workspace]
members = [
  "crates/stdarch-verify"
```
This didn't show up when testing with `-Zbuild-std` since the [standard list of crates](https://github.com/rust-lang/cargo/blob/f624095e1c98228a74a165ddb702078c0dd8b81e/src/cargo/core/compiler/standard_lib.rs#L26-L30) to include when building `std` does not include `stdarch`, but it will show up if a user explicitly requests `stdarch`. Or, perhaps more importantly, because of https://github.com/rust-lang/rust/issues/95736, many editors (like IntelliJ) won't treat the root of `rust-src` as a workspace, and will instead recurse into all the sub-crates directly, which then includes `stdarch`.

Reply:
Right, my bad.

Reply:
I filed https://github.com/rust-lang/rust/pull/97228 with a fix 👍